MONITORING_IMPLEMENTATION_v2.3.md

```
@@ -0,0 +1,426 @@
+   cd coolify-monitoring
+   ./scripts/start-monitoring.sh
+   ```
+
+2. **Open Ports on Server** - Via Admin UI:
+   - Login: `https://your-domain.com/admin`
+   - Navigate: Monitoring → Port Configuration
+   - Click: "Open Port" for services
+   - Choose duration (15-240 minutes)
+
+3. **View Dashboards**:
+   - Prometheus: http://localhost:9090
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)